java/src/jmri/jmrit/timetable/TimeTableBundle_nl.properties

```diff
@@ -148,7 +148,7 @@ StartTimeError = Train start time is not between {0}:00 and {1}:00
 NumberFormatError = "{0}" is not a valid number for {1}
 SegmentChangeError = Unable to find station "{0}" in segment "{1}"
 ImportCompleted = The {0} import has completed.\nThe changes have been saved.\nClose and re-open the Timetable to refresh.
-ImportFailed = The {0} import has failed.\nCheck the JMRI System console for details.
+ImportFailed = The {0} import has failed.\nCheck the JMRI System Console for details.
```

Comment:
Just a comment in passing, not so much about this change:  We should try to standardize on how we add new properties.  When adding a new property `TranslatableFoo`, should we just put it in the default .properties file?  Other languages will pick it up from there, and we don't have to worry about consistent edits then.  Or should we put the same (English) line in all the other translated .properties file?  That clues a translator that there's an item to translate.